src/main/java/com/spacesongr/songr/Controllers.java

```diff
@@ -0,0 +1,35 @@
+package com.spacesongr.songr;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.ArrayList;
+
+@Controller
+public class Controllers {
+    @GetMapping("/hello")
+    public String getGreeting(){
+        return "./resources/templates/hello_world.html";
+    }
+
+    @GetMapping("/capitalize/{capitalize}")
+    public String getcapitalizedTerm(Model m, @PathVariable String toCapitalize){
```

Comment:
In order to use `@PathVariable`, the variable name has to be *exactly* the same as the query string you're passing in (the part in the curly brackets). In this case, PathVariable is looking for a variable called `toCapitalize`, but in your query string, you've named the variable `capitalize`. To fix this, either change your get mapping to `"/capitalize/{toCapitalize}" or rename your PathVariable as `capitalize`.